src/generation/generate.rs

```diff
@@ -6354,19 +6354,20 @@ fn gen_comment(comment: &Comment, context: &mut Context) -> Option<PrintItems> {
 
   return Some(match comment.kind {
     CommentKind::Block => {
-      if is_js_doc(&comment.text) {
-        gen_js_doc(comment, context)
+      if is_js_doc_or_multiline_block(&comment.text) {
+        gen_js_doc_or_multiline_block(comment, context)
       } else {
+        // Single-line comment block
         ir_helpers::gen_js_like_comment_block(&comment.text)
       }
     }
     CommentKind::Line => ir_helpers::gen_js_like_comment_line(&comment.text, context.config.comment_line_force_space_after_slashes),
   });
 
-  fn is_js_doc(text: &str) -> bool {
+  fn is_js_doc_or_multiline_block(text: &str) -> bool {
     // be strict about what a js doc is for now
-    if text.starts_with('*') && text.contains('\n') {
-      for line in text.trim().split('\n').skip(1) {
```

Comment:
`skip(1)` was breaking on certain block comments that started with a leading newline, e.g.
```
/*
 test */
```
because the `trim()` (strip \n + leading whitespace) followed by the the `skip(1)` skipping the entire comment would then default to `true`. 

Removing it had no impact on tests, so I'm assuming it was just added in the first place as a optimisation, since the trimmed first line of a JSDoc is always just '*'.